### PR TITLE
Update requirements and README.md

### PR DESCRIPTION
**Pull request by moshe-shahar**
Update requirements to support using the latest manifest-tool and fix the README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,31 +19,33 @@ $ pip install -I pelion_test_lib*.whl
 
 ## Basic usage
 
-- Build the [Device Management Client example application](https://www.pelion.com/docs/device-management/current/connecting/device-management-client-tutorials.html) for your board and flash it.
-- The [API key](https://www.pelion.com/docs/device-management/current/integrate-web-app/api-keys.html) is used from Mbed's default `CLOUD_SDK_API_KEY` environment variable, but you can override it by defining a separate variable using a command-line command:
-    - Linux: `export PELION_CLOUD_API_KEY=[api_key_here]`
-    - Windows: `set PELION_CLOUD_API_KEY=[api_key_here]`
+- Build the [Device Management Client example application](https://developer.pelion.com/docs/device-management/current/connecting/mbed-os.html) for your board and flash it.
+- Set `PELION_CLOUD_API_KEY` environment variable with your [access key](https://developer.pelion.com/docs/device-management/current/user-account/application-access-keys.html).
```
Comment (teetak01) [inline README.md:23]: We probably could deprecate these API_KEY macros and only discuss about access keys.

API keys are deprecated.

**Comment by moshe-shahar** [comment]
@calmjm , @GuyWi, please review.

**Review by teetak01 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (moshe-shahar) [reply README.md:23]: this requires changes in the library which I prefer not to handle right now.

Comment (teetak01) [reply README.md:23]: But I think this can be seperate PR anyways.

Comment (moshe-shahar) [reply README.md:23]: Let's leave that for SyTE to decide and handle.

**Comment by moshe-shahar** [comment]
I run a test with those changes internally using MCCE CI.

**Review by GuyWi — APPROVED** [review]

**Review by calmjm — APPROVED** [review]

**Comment by moshe-shahar** [comment]
@calmjm , thank you!
Can you please merge and tag it so I can update our apps?

**Pull request merged**